### ISSUE TITLE
DB Error - Dublicate entry for key 'PRIMARY'

### ISSUE THREAD

**Issue by rabuzarus** [friendica/friendica#4816]
### Actual behavior
Following DB Error does appear in my logs
```
dba.php:536:e  DB Error 1062: Duplicate entry '84-Friendica' for key 'PRIMARY'
dba::insert, PortableContact::lastUpdated, DiscoverPoCo::execute, Worker::processQueue
INSERT INTO `gserver-tag` (`gserver-id`, `tag`) VALUES ('84', 'Friendica')
```

### Friendica version you encountered the problem
2018-05-dev (latest develop)

### Friendica source (git, zip)
git

### PHP version
5.6

### SQL version
mysql  Ver 15.1 Distrib 10.0.32-MariaDB

**Comment by annando** [comment]
I have a fix for it. I guess I will submit the PR later this evening.

**Comment by MrPetovan** [comment]
Closed via #4822
